Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -363,6 +405,7 @@ function Configure-ConditionalAccessPolicyForPIM
 
     if (-not $DryRun)
     {
+        # It is a wet run, hence we need to create policy assignments for the roles for which CA policy is not there.
         try
         {
             $policyAssignments = Get-AzRoleManagementPolicyAssignment -Scope "subscriptions/$($SubscriptionId)"
```

Comment:
Please check are we making this call 2 times?

Reply:
Yeah, in some cases it could have been two calls, removed one of them and placed one at the top.